### PR TITLE
Browser defaults from env

### PR DESCRIPTION
**Pull request by MilosKozak**

### CODE REVIEW

File: env.js [hunk 1]
```diff
@@ -46,6 +46,78 @@ function config ( ) {
...
   env.enable = readENV('ENABLE');
+
+  env.defaults = { // currently supported keys must defined be here
+	  'units': 'mg/dL'
+	, 'timeFormat': '12'
+	, 'nightMode': false
+	, 'showRawbg': 'never'
+	, 'customTitle': 'Nightscout'
+	, 'theme': 'default'
+    , 'alarmUrgentHigh': true
```
Comment (jasoncalabrese) [inline env.js]: Check whitespace, looks like mix of tabs and space.  It bugs me that all the files use different whitespace conventions, I'd reformat it all, but don't want to deal with the merge conflicts

File: env.js [hunk 2]
```diff
@@ -46,6 +46,78 @@ function config ( ) {
...
+    , 'alarmHigh': true
+    , 'alarmLow': true
+    , 'alarmUrgentLow': true
+	, 'language': 'en' // not used yet
+	} ;
+	
+  ///////////// Single line envs part. Can be removed if necessary
+  // parse defaults here, expect format `key1=value1 key2=value2` split on space, ignore commas
+  var rawdefaults = readENV('DEFAULTS','');
```
Comment (jasoncalabrese) [inline env.js]: don't think we need this parsing anymore, should also stop using `DEFAULTS`

Comment (MilosKozak) [inline env.js]: ok. if you cannot leave it there for me merge without marked paragraph pls :)

Comment (MilosKozak) [inline env.js]: readme is updated to describe per line values

**Comment by MilosKozak** [comment]
removed single line defaults
rebased to current dev
resolved conflicts

**Comment by jasoncalabrese** [comment]
Thanks @MilosKozak this will make some people happy

**Pull request merged**